styles/styles.css

```diff
@@ -13,7 +13,8 @@
   /* fonts */
   --body-font-family: var(--udexTypographyBodyMFontFamily);
   --heading-font-family: var(--udexTypographyHeadingRegularMFontFamily);
-  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
+  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono',
+    monospace;
```

Comment:
Shouldn't this be removed? This should come from the theme styles.

Reply:
Quite frankly most of it is boilerplate code which conflicts with SAP styles. I'll try to slowly phase it out and move much of it to theme.